src/main.py

```
@@ -1,253 +1,206 @@
+            except Exception as e:
+                return Response.json({"error": f"Failed to load UI template: {e}"}, status=500)
+
+        # ZIP-only submission
+        if request.method == "POST" and url.pathname == "/submit":
             ip = get_client_ip(request)
-            
+
             try:
                 payload = await request.json()
-            except:
+            except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)